src/clj_3df/compiler.cljc

```diff
@@ -240,10 +240,10 @@
   (bound-symbols [this]
     (if (some? binding) (bound-symbols binding) args))
```

Comment:
Unrelated to this PR, but this probably is not fully correct yet. At the very least, `result-sym` should be part of the bound symbols. But we also have to think about whether the ouput binding is supposed to replace the input binding, or whether they co-exist, etc...

Reply:
Yep good point. I'll have a look.

Reply:
I'll create an issue.